nodes/api-common/Cargo.toml

```diff
@@ -22,10 +22,13 @@ pprof = { default-features = false, features = [
   "protobuf-codec",
 ], optional = true, version = "0.15" }
 serde = { default-features = false, features = ["alloc", "derive"], version = "1.0" }
+serde_json = { workspace = true, optional = true }
 serde_with = { default-features = false, features = ["macros"], workspace = true }
 tokio = { default-features = false, features = ["time"], optional = true, version = "1" }
 tower_governor = { default-features = false, features = ["axum"], version = "0.3" }
 tracing = { default-features = false, optional = true, workspace = true }
+zksign = { workspace = true, optional = true }
 
 [features]
 profiling = ["pprof", "tokio", "tracing"]
+wallet    = ["serde_json", "zksign"]
```

Comment:
can we drop the feature gate?

Reply:
I just checked, adding this wallet feature flag would have added 50 new combinations to check in cargo hack 🤯. I think we really should avoid flags unless there's some strong need to compile some code out, it's killing the CI runtimes